spec/requests/admin/web_hooks_controller_spec.rb

```diff
@@ -57,10 +59,9 @@
 
     describe '#ping' do
       it 'enqueues the ping event' do
-        Jobs.expects(:enqueue)
-          .with(:emit_web_hook_event, web_hook_id: web_hook.id, event_type: 'ping', event_name: 'ping')
-
-        post :ping, params: { id: web_hook.id }, format: :json
+        expect do
+          post "/admin/api/web_hooks/#{web_hook.id}/ping.json"
+        end.to change { Jobs::EmitWebHookEvent.jobs.size }.by(1)
```

Comment:
We need to test for the `job_args` of the new job as well.

Reply:
Done ✅